src/async_fuse/memfs/node.rs

```diff
@@ -209,10 +215,11 @@ impl DefaultNode {
             attr,
             data,
             fd,
-            // lookup count set to 1 by creation
-            open_count: AtomicI64::new(1),
+            // open count set to 0 by creation
+            open_count: AtomicI64::new(0),
             // open count set to 1 by creation
             lookup_count: AtomicI64::new(1),
+            deferred_deletion: AtomicBool::new(false),
```

Comment:
If all the operation on this field is protected by a lock, we can use bool directly

Reply:
Yes. It is possible. But I want to keep for two reasons. 1. make API as &self instead of &mut self. 2. it is possible to get read lock instead of write log to mark deferred_deletion.

Reply:
Good point